cfgov/unprocessed/css/enhancements/base.less

```diff
@@ -19,11 +19,7 @@
 */
 
 body {
-    .webfont-regular();
-
-    .respond-to-max( @bp-sm-max, {
-        overflow-x: hidden;
-    } );
+    .webfont-regular(); // Moved to CF
```

Comment:
IDK, I was going to try and go through and document what is and isn't in CF so that whoever ends up making the change over knows what they can safely delete without having to go through everything. Does that make sense?